linter/linter.go

```diff
@@ -581,6 +605,24 @@ func (l *Linter) lintSubRoutineDeclaration(decl *ast.SubroutineDeclaration, ctx
 	return types.NeverType
 }
 
+func (l *Linter) lintPenaltyboxDeclaration(decl *ast.PenaltyboxDeclaration) types.Type {
+	// validate penaltybox name
+	if !isValidName(decl.Name.Value) {
+		l.Error(InvalidName(decl.Name.GetMeta(), decl.Name.Value, "penaltybox").Match(PENALTYBOX_SYNTAX))
+	}
+
```

Comment:
Should we lint the block statement be empty? If so, we may add more rule for that.